.github/workflows/ci_build.yml

```diff
@@ -8,13 +8,24 @@ on:
 
 jobs:
   build:
-    runs-on: windows-2019
+    runs-on: windows-2022
 
     steps:
-      - uses: actions/checkout@v2
+      - name: Check Out
+        uses: actions/checkout@v3.0.2
+
+      - name: Install .NET 6
```

Comment:
Install the latest net6/netcoreapp3.1, also updated the github actions to the latest versions.

Recommend you maybe consider using dependabot to get latest github versions (along with .net components)

To reduce spam I generally have Dependabot only check for updates once a month.

Reply:
Thanks for the tip. From the looks of it, Dependabot is just a config file in the repo so if you'd like, I'd be happy to receive the correct config as a PR.

Reply:
Would you like it as part of this pr or separate?

Reply:
A separate ticket and PR please. 

And I've just noticed, the CI build does do nuget pack, so there's a problem with the .NET 3.5 target framework stemming from https://github.com/rubicon-oss/CoreRhinoMocks/pull/5#discussion_r905664101:
https://github.com/rubicon-oss/CoreRhinoMocks/runs/7060597905?check_suite_focus=true#step:10:35

Reply:
I'll check out the pack tomorrow

Reply:
meanwhile: I'm currently updating the workflow so we can ci test the build process. should be integrated in tomorrow's develop.